web/styles/subscriptions.css

```diff
@@ -1033,24 +1033,9 @@ h4.user_group_setting_subsection_title {
             display: flex;
 
             .subscribe-button,
-            .join_leave_button,
```

Comment:
I did not notice this before, but the "Join/Leave" button should also have this CSS for height and margin.